src/ensembl/src/content/app/browser/browser-nav/BrowserNavBar.tsx

```diff
@@ -20,9 +20,7 @@ type StateProps = {
 
 type DispatchProps = {};
 
-type OwnProps = {
-  browserElement: HTMLDivElement;
-};
+type OwnProps = {};
```

Comment:
Do we need the above two definitions?

Reply:
I think this is @imran-ebi 's preferred way of writing prop types :-) I prefer to just lump everything in a single Props type. Am happy to get rid of these empty types (but then I would rename StateProps to Props).